www/js/draw-model.js

```diff
@@ -568,6 +568,9 @@ define([
 				if(connection.sourceId == nodeID || connection.targetId == nodeID){
 					try{
 						this._instance.detach(connection);
+						var lostNode = connection.sourceId === nodeID ? connection.targetId : connection.sourceId;
+						//if the updated connection points to an alien node (no authoredID), then update the candelete field and delNodeTag which facilitates right click delete
+						this.updateDeleteLabel(lostNode);
```

Comment:
(No change required here, just a comment.)

It seems a little odd the draw-model would have to do this update instead of it being done in the model code (as the model should contain all the info required to do this update by itself).  But I guess that's because our model and view are too tightly coupled already.  Changing that would be a much, much bigger refactoring that we don't have time for right now, but let's keep it in mind for the future.